packages/web-components/src/components/logo-grid/logo-grid.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { css, customElement, html, property } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './logo-grid.scss';
+import DDSContentBlock from '../content-block/content-block';
+import '../content-block/content-block-heading';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Logo grid.
+ *
+ * @element dds-logo-grid
+ */
+@customElement(`${ddsPrefix}-logo-grid`)
+class DDSLogoGrid extends StableSelectorMixin(DDSContentBlock) {
+  static get stableSelector() {
+    return `${ddsPrefix}--logo-grid`;
+  }
+
+  static get hideBorderClass() {
+    return `${prefix}--logo-grid__no-border`;
+  }
+
+  @property({ attribute: 'hide-border', reflect: true, type: Boolean })
+  hideBorder = false;
+
+  protected updated() {
+    this.classList.toggle((this.constructor as typeof DDSLogoGrid).hideBorderClass, this.hideBorder);
+  }
```

Comment:
We don't use CSS classes on our own Web Components, given it tends to clash with CSS classes application code sets. We use attribute selectors instead:

```suggestion
```